src/js/edu-benefits/1995/config/form.js

```diff
@@ -0,0 +1,66 @@
+import { fullName, ssn } from '../../../common/schemaform/definitions';
+import { validateSSN } from '../../../common/schemaform/validation';
+import IntroductionPage from '../components/IntroductionPage';
+
+const formConfig = {
+  urlPrefix: '/1995/',
+  introduction: IntroductionPage,
+  confirmation: null,
+  chapters: {
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          path: 'applicant-information',
```

Comment:
I wasn't sure since the title of the first page is Applicant information in the GH issue for the first page.